Author: Pino Toscano <pino@debian.org>
Reviewed-by: Michael Banck <mbanck@debian.org>
Description: Fix FTBFS because of missing files
 FTBFS on hurd-i386: missing .desktop file and its icon
Bug-Debian: http://bugs.debian.org/659048
Forwarded: yes

--- a/avogadro/src/CMakeLists.txt
+++ b/avogadro/src/CMakeLists.txt
@@ -169,10 +169,10 @@
 endif(APPLE AND ENABLE_AVO_PACKAGE)
 
 # Install a .desktop file and icon for Linux users
-if (CMAKE_SYSTEM_NAME MATCHES Linux OR CMAKE_SYSTEM_NAME MATCHES kFreeBSD)
+if (CMAKE_SYSTEM_NAME MATCHES Linux OR CMAKE_SYSTEM_NAME MATCHES kFreeBSD OR CMAKE_SYSTEM_NAME STREQUAL GNU)
   install(FILES "avogadro.desktop" DESTINATION ${XDG_APPS_INSTALL_DIR})
   install(FILES "icons/avogadro-icon.png" DESTINATION ${XDG_ICON_INSTALL_DIR})
-endif(CMAKE_SYSTEM_NAME MATCHES Linux OR CMAKE_SYSTEM_NAME MATCHES kFreeBSD)
+endif(CMAKE_SYSTEM_NAME MATCHES Linux OR CMAKE_SYSTEM_NAME MATCHES kFreeBSD OR CMAKE_SYSTEM_NAME STREQUAL GNU)
 
 #add_subdirectory(projectplugins)
 
